src/developer/web-api/tracker.md

```diff
@@ -1880,6 +1879,7 @@ Returns a list of events based on filters.
 |`trackedEntity`|`String`|`uid`| Identifier of tracked entity instance|
 |`enrollment`|`String`|Comma-delimited list of `uid`| Filter the result down to a limited set of IDs by using enrollment=id1;id2.|
 |`includeDeleted`|`Boolean`| |  When true, soft deleted events will be included in your query result.|
+|`order`|`String`|comma-delimited list of `OrderCriteria` in the form of `propName:sortDirection`.<br><br>Available properties are: `completedAt`, `createdAt`, `createdAtClient`, `enrolledAt`, `updatedAt` and `updatedAtClient` <br><br> Example: `createdAt:desc`<br><br>**Note:** `propName` is case sensitive, `sortDirection` is case insensitive. `sortDirection` defaults to `asc` when non provided.|Sort the response based on given `OrderCriteria`|
```

Comment:
see https://github.com/dhis2/dhis2-docs/pull/1272#issuecomment-1881096789 Please use the blueprint. We should not include any Java classes in user docs. Unless the docs are about a Java SDK or are Javadocs 😋